src/diy/smol-slime.md

```diff
@@ -13,16 +13,17 @@ Interested, have questions, or issues with this project? Chat with us in ***#smo
+| eByte Dongle (E104-BT5040U)                     | Cheapest Receiver, free shipping from AliExpress, and has PCB Trace Antenna. <br /> Sold on Alibaba by `Chengdu Ebyte Electronic Technology Co., Ltd.` <br/> - `E104-BT5040U` is compatible with all the programs of Nordic's original nRF52840 USB Dongle, and IO ports and hardware resources. <br/> - `E104-BT5040UA` is not usable as receiver because it's built to capture BLE4.2 and BLE5.0 protocol packets only. | Has a built-in PCB antenna. It is easier to use a USB extender cable to get better signal strength than to modify the antenna.                 |
+| Nordic Semiconductor nRF52840 Dongle (PCA10059) | More expensive, not free shipping from Digikey/Mouser, and has PCB Trace Antenna.                                                                                                                                                                                                                                                                                                                                         | Has a built-in PCB antenna. It is easier to use a USB extender cable to get better signal strength than to modify the antenna.                 |
+| SuperMini nRF52840                              | Cheapest option, but having a ceramic antenna and your trackers also having a ceramic antenna will reduce signal strength and range.                                                                                                                                                                                                                                                                                      | When used as a receiver, signal strength can be improved by adding a 31cm wire to the antenna. The stock antenna is insufficient for receiver. |
+| Seeed Studio XIAO nRF52840                      | Expensive option, but smaller.                                                                                                                                                                                                                                                                                                                                                                                            | When used as a receiver, signal strength can be improved by adding a 31cm wire to the antenna. The stock antenna is insufficient for receiver. |
+### 🏃 Tracker
+When choosing trackers, it is also important to consider [how many trackers you may need](../slimevr101.md#how-many-trackers-do-you-need).  
```

Comment:
This wording is a bit vague. I assume you mean "please refer to this page to determine how many trackers you will need and choose the tracker based off this" but as it is written here it is very ambiguous to why you would need to refer to this page.